automation/carbonite-too-big-check.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -euo pipefail
+
+#VAULT_TOKEN=$(cat /etc/vault-token-dsde)
+WORKFLOW_ID="8fcff291-f4aa-401a-bf16-70e7fa76e01d"
+EXPECTED_STATUS="TooLargeToArchive"
+
+if [[ -z "${WORKFLOW_ID}" ]]
+then
+  echo "Missing required env input: WORKFLOW_ID"
```

Comment:
Kinda looks like there was a change from default parameters to required parameters that was meant to be cleaned up? 🙂